src/services/content/hooks/index.js

```diff
@@ -1,17 +1,17 @@
 const { authenticate } = require('@feathersjs/authentication');
 const globalHooks = require('../../../hooks');
-
+const { disallow } = require('feathers-hooks-common');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`feathers-hooks-common` import should occur before import of `../../../hooks`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5367171)